### PR TITLE
Fix: Bump sigs.k8s.io/controller-runtime from 0.15.1 to 0.16.3 

### PR DESCRIPTION
**Pull request by cool-wangtongzhou**
update-apimachinery-0.28.3

### CODE REVIEW

File: pkg/k8s/crd/main.go [hunk 1]
```diff
@@ -79,9 +79,9 @@ func main() {
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
+		Scheme: scheme,
+		// MetricsBindAddress:     metricsAddr,
+		// Port:                   9443,
```
Comment (tomqin93) [inline pkg/k8s/crd/main.go:84]: remove if not used

File: pkg/deviceshifu/deviceshifutcp/etc/edgedevice/config/driverProperties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+driverSku: Edgenesis Mock Device
+driverImage: edgenesis/mockdevice:v0.0.1
```
Comment (tomqin93) [inline pkg/deviceshifu/deviceshifutcp/etc/edgedevice/config/driverProperties:2]: what are these for?

File: pkg/deviceshifu/deviceshifutcp/etc/edgedevice/config/instructions [hunk 3]
```diff
@@ -0,0 +1,2 @@
+instructions:
+  cmd:
```
Comment (tomqin93) [inline pkg/deviceshifu/deviceshifutcp/etc/edgedevice/config/instructions:2]: what are these for?

File: pkg/deviceshifu/deviceshifutcp/etc/edgedevice/config/telemetries [hunk 4]
```diff
@@ -0,0 +1,2 @@
+telemetrySettings:
+  telemetryUpdateIntervalInMilliseconds: 1000
```
Comment (tomqin93) [inline pkg/deviceshifu/deviceshifutcp/etc/edgedevice/config/telemetries:2]: what are these for?

**Review by tomqin93 — CHANGES_REQUESTED** [review]
@cool-wangtongzhou added some comment, please take a look

**Comment by cool-wangtongzhou** [comment]
> @cool-wangtongzhou added some comment, please take a look
I have deleted unnecessary things, you can take a look.

File: pkg/k8s/crd/main.go [hunk 5]
```diff
@@ -79,9 +79,8 @@ func main() {
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
+		Scheme: scheme,
+
```
Comment (tomqin93) [inline pkg/k8s/crd/main.go:83]: remove empty line

**Review by tomqin93 — APPROVED** [review]
LGTM

**Comment by tomqin93** [comment]
fixed in https://github.com/Edgenesis/shifu/pull/738, closing this PR

**Pull request closed without merging**